src/steamship/data/tags/tag.py

```diff
@@ -122,29 +113,194 @@ def query(
         )
         return res
 
-    # @staticmethod
-    # def search(
-    #     client: Client,
-    #     search_index: Any,
-    # ) -> TagQueryResponse:
-    #     # Avoid the circular import.
-    #     from steamship.data.plugin.index_plugin_instance import EmbeddingIndexPluginInstance
-    #
-    #     if not isinstance(search_index, EmbeddingIndexPluginInstance):
-    #         raise SteamshipError(message="The search_index field")
-    #
-    #
-    #     req = TagQueryRequest(tag_filter_query=tag_filter_query)
-    #     res = client.post(
-    #         "tag/query",
-    #         payload=req,
-    #         expect=TagQueryResponse,
-    #     )
-    #     return res
+
+class TimestampTag(Tag):
+    def __init__(
+        self,
+        start_time_s: float,
+        end_time_s: float,
+        start_idx: Optional[int] = None,
+        end_idx: Optional[int] = None,
+        value: Optional[Dict[str, Any]] = None,
```

Comment:
This is definitely a much more pleasant interface for tag creation!

Reply:
Agree -- this is really nice.